Rohit/PostgresAssignment/src/controller/class.ts

```diff
@@ -0,0 +1,50 @@
+import { Context } from "vm";
+import { classSchema } from "../validation/schema";
+
+const classController = require("../services/class");
+
+interface IClassInfo {
+  classId: number;
+  stId: number;
+}
+
+exports.getClassInfo = async (ctx: Context) => {
+  try {
+    let [rows]: Array<{ rows: IClassInfo }> = [];
+    rows = await classController.get_classes();
+
+    ctx.response.status = 200;
+    ctx.response.type = "application/json";
+    ctx.body = rows.rows;
+  } catch (err) {
+    ctx.response.status = 400;
```

Comment:
why 4XX ? should it be 5XX ??? ref: https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment

Reply:
resolved